index.html

```diff
@@ -14,5 +15,24 @@ <h4 class="nav-logo">nathaniel</h4>
 				<a class="nav-link" href="#contact">Contact</a>
```

Comment:
will you be adding more here or just contacts? 🤔

Reply:
just projects and contact for now, maybe more once site starts to grow